draft-birkholz-scitt-architecture.md

```diff
@@ -298,32 +298,20 @@ For example, consider a TS implemented using a set of replicas, each running wit
 
 ### Registration Policies
 
-> **Editor's note**
->
-> The initial version of this document assumes Registration policies are set for the lifetime of the Registry, and that they apply to all Issuers and Feeds uniformly.
-> There is an ongoing discussion on how to make the design more flexible to allow per-Issuer and per-Feed Registration policies, and whether such policies should be updatable or if a policy change requires a Feed change.
-> Please contribute your comments to the SCITT mailing list.
-
-Each TS is initially configured with a set of Registration policies, which will be applied for the lifetime of the Registry.
-A Registration policy represents a predicate that takes as input the current Registry and the Envelope of a new Claim to register (including the `reg_info` header which contains customizable additional attributes), and returns a Boolean decision on whether the Claim should be included in the Registry or not. A TS MUST ensure that all its Registration policies return a positive decision before adding a Claim to the Registry.
-
-While Registration policies are a burden for Issuers (some may require them to maintain state to remember what they have signed before) they support stronger transparency guarantees, and they greatly help Verifiers and auditors in making sense of the information on the Registry. (This is particularly relevant for parties that verify Receipts on their own, without accessing the Registry). For instance, if a TS doesn't apply any policy, Claims may be registered in a different order than they have been issued, and old Claims may be replayed, which makes it difficult to understand the logical history of an Artifact, or to prevent rollback attacks.
+A TS that accepts to register any valid claim offers limited value to verifiers: the baseline transparency guarantee ensures completeness of audit, which can help detect equivocation.
```

Comment:
```suggestion
A TS that accepts to register any valid claim offered by an issuer would end up providing only limited value to verifiers. In consequence, a baseline transparency guarantee policing the registration of claims is required to ensure completeness of audit, which can help detect equivocation.
```

Reply:
Agree with the first change but not with the second - audit completeness is the only property that *does not* depend on the registration policy, just the fact that the claim is in the append-only registry.

```suggestion
A TS that accepts to register any valid claim offered by an issuer only provides the baseline audit completeness of transparency to verifiers., which may help detect equivocation but is not enough to filter and police bad claims.
```